### ISSUE TITLE
A11y_Visual Studio Code Services_API References_Theme color_Info and Relationships:Two Navigation landmark with same name, no unique accessible name is defined for the landmark

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/vscode-docs#6298]
### GitHub Tags

#WCAG1.3.1; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023; #Info and Relationships;

### Actual Result
Two Navigation landmark with same name, no unique accessible name is defined for the landmark.

### Expected Result
Two Navigation landmark with same name should not be provide unique accessible name should be defined for the landmark.

### Environment Details

Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 113.0.5672.93 (Official Build) (64-bit)

### Repro Steps

1. Hit the application URL: https://code.visualstudio.com/
2. Tab till "API" link and press enter.
3. Tab till "References" link in the left navigation and press enter.
4. Tab till "Theme color" link which is present under the reference and press enter.
5. Verify whether the two Navigation landmark with same name is provided or not.

### User Impact
Screen reader users will get confused if unique name is not defined for the landmarks. 

### Attachment
![image](https://github.com/microsoft/vscode-docs/assets/99243079/f48cba5f-1caf-433f-8b78-3b4c03477e94)

**Comment by rzhao271** [comment]
Ref https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/navigation.html

**Comment by Samhitha66** [comment]
@rzhao271 
As per the document 
If a navigation landmark has an identical set of links as another navigation landmark on the page, use the same label for each navigation landmark.
In our case we have same navigation name even though we have different set of links.
Please provide your input.
#Regressed:05-17-23;

**Comment by rzhao271** [comment]
@Samhitha66 that comment was a note to myself. I noticed that some nav sections don't have aria labels, so I can fix that. However, the top nav should have an aria label that says Top Navigation. NVDA reads that aria label for me. Which screenreader are you using?

**Comment by Samhitha66** [comment]
Regressed the issue in the below environment
URL:https://code.visualstudio.com/api/references/theme-color
The issue is fixed. Hence closing the bug.
Please find the attachment for the reference.
#Regressed:05-25-23; #Closed;
![image](https://github.com/microsoft/vscode-docs/assets/99243079/f6522469-b105-4cc1-9188-82ec5976b46e)
